test/functional/page_objects/visualize_chart_page.ts

```diff
@@ -391,10 +411,13 @@ export function VisualizeChartPageProvider({ getService, getPageObjects }: FtrPr
       );
     }
 
-    public async openLegendOptionColors(name: string, chartSelector = elasticChartSelector) {
+    public async openLegendOptionColors(name: string, chartSelector = xyChartSelector) {
       await this.waitForVisualizationRenderingStabilized();
       await retry.try(async () => {
-        if (await this.isVisTypeXYChart()) {
+        if (
+          (await this.isNewLibraryChart(xyChartSelector)) ||
+          (await this.isNewLibraryChart(pieChartSelector))
```

Comment:
what if you just use 
```suggestion
          (await this.isNewLibraryChart(chartSelector))
```

Reply:
Fixed https://github.com/elastic/kibana/commit/9994cedc5d135bbc137f8458f11d661be81d43f3